src/Tests/ArrayKeyVisitor.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ *
+ * EPV :: The phpBB Forum Extension Pre Validator.
+ *
+ * @copyright (c) 2014 phpBB Limited <https://www.phpbb.com>
+ * @license       GNU General Public License, version 2 (GPL-2.0)
+ *
+ */
+namespace Phpbb\Epv\Tests;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\Array_;
+use PhpParser\Node\Expr\ArrayItem;
+use PhpParser\Node\Scalar\String_;
+use PhpParser\NodeVisitorAbstract;
+
+class ArrayKeyVisitor extends NodeVisitorAbstract
+{
+	/**
+	 * @var array
+	 */
+	private $keys;
+
+	/**
+	 * @param array $nodes
+	 * @return void|null|Node[]
+	 */
+	public function beforeTraverse(array $nodes)
+	{
+		$this->keys = [];
+	}
+
+	/**
+	 * @param Node $node
+	 * @return void|null|int|Node
+	 */
+	public function enterNode(Node $node)
+	{
+		if ($node instanceof Array_)
+		{
+			foreach ($node->items as $item)
+			{
+				/** @var ArrayItem $item */
+				if ($item->key instanceof String_)
+				{
+					$this->keys[] = $item->key->value;
+				}
+			}
+		}
```

Comment:
there is no `else` to output Err0r message using this `$this->output->addMessage(...)`.
Like in line : https://github.com/phpbb/epv/pull/96/files#diff-e1b04a6070a8aea621fb26a83bd723de8f86c5957aef5b1952a9910c0e0e84a5L148